From 31432c67916bbb3684d3e821c4eeaec9dc4de7a1 Mon Sep 17 00:00:00 2001
From: Stefan Tatschner <stefan@sevenbyte.org>
Date: Fri, 21 Nov 2014 13:05:36 +0100
Subject: [PATCH] Removed titleColSpan of uncategorized topics

This behaviour is not consistent any more. We can toggle "suppress
uncategorized badge" in the admin page which has this description:
"Don't show the badge for uncategorized topics in topic lists."

So I think the intended behaviour is "show the category", or "don't show
the category". Not showing the badge on pinned topics is quite confusing
when it's explicitly enabled in the admin settings.

There is also a thread an meta.discourse.org:
https://meta.discourse.org/t/22245
---
 app/assets/javascripts/discourse/controllers/topic-list-item.js.es6 | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/app/assets/javascripts/discourse/controllers/topic-list-item.js.es6 b/app/assets/javascripts/discourse/controllers/topic-list-item.js.es6
index e5128e2..79b4397 100644
--- a/app/assets/javascripts/discourse/controllers/topic-list-item.js.es6
+++ b/app/assets/javascripts/discourse/controllers/topic-list-item.js.es6
@@ -23,9 +23,7 @@ export default Ember.ObjectController.extend({
   }.property('controllers.discovery/topics.selected.length'),
 
   titleColSpan: function() {
-    // Uncategorized pinned topics will span the title and category column in the topic list.
-    return (!this.get('controllers.discovery/topics.hideCategory') &&
-             this.get('model.isPinnedUncategorized') ? 2 : 1);
+    return (!this.get('controllers.discovery/topics.hideCategory'));
   }.property('controllers.discovery/topics.hideCategory', 'model.isPinnedUncategorized'),
 
   hideCategory: function() {
